docs/source/install.rst

```diff
@@ -425,3 +425,41 @@ You can specify a comma-separated list of ISAs if you have multiple GPUs of diff
 
   If you don't specify the ``HCC_AMDGPU_TARGET`` environment variable, CuPy will be built for the GPU architectures available on the build host.
   This behavior is specific to ROCm builds; when building CuPy for NVIDIA CUDA, the build result is not affected by the host configuration.
+
+Limitations
+-----------
+
+The following CUDA-specific features are not available in ROCm:
+
+* CUDA Array Interface
+* cuTENSOR
+* Atomic addition in FP16 (``cupy.ndarray.scatter_add`` and `cupyx.scatter_add`)
+* Multi-GPU FFT and FFT callback
+* Some random number generation algorithms
+
+The following features are not yet supported:
+
+* Several options in RawKernel/RawModule APIs: Jitify, dynamic parallelism
+* Sparse matrices (``cupyx.scipy.sparse``)
+* cuDNN (hipDNN)
+* Hermitian/symmetric eigenvalue solver (``cupy.linalg.eigh``)
+* Polynomial roots (uses Hermitian/symmetric eigenvalue solver)
+
+The following features may not work in edge cases (e.g., some combinations of dtype):
+
+.. note::
+   We are investigating the root causes of the issues. They are not necessarily
+   CuPy's issues, but ROCm may have some potential bugs.
+
+* 32-bit CArray boundary (HIP is known to fail with sizes 2**32-1024)
+* ``cupy.ndarray.__getitem__``
+* ``cupy.cuda.cufft.PlanNd``
```

Comment:
I think this is currently private (despite I'd love to make it public)?

Reply:
Thanks! I'll remove the line.